submissions/Roka20012/js-dom/interactive.js

```diff
@@ -0,0 +1,81 @@
+"use strict";
+
+const nav = document.getElementById("navbar");
+const text = document.getElementById("text");
+const burger = document.getElementsByClassName("burger2")[0];
+/**
+ * закриває меню якщо ми натискаємо на пункт меню 
+ * коли ширина екрана менша 900px
+ * author: Rostyk Broslavsky
+ * email: broslavsky.rostislaw@gmail.com
+ */
+function hideBurgerMenu(target) {
+    let computedStyle = getComputedStyle(burger);
+
+    if (computedStyle.left === "15px") {
+        nav.classList.toggle("menu-open");
+        burger.classList.toggle("open");
+    }
+}
+
+/**
+ * додає колір для вибраного елемента
+ * author: Rostyk Broslavsky
+ * email: broslavsky.rostislaw@gmail.com
+ */
+function setColorForElement(target) {
+
+    [...nav.children].forEach(el => {
+        el.classList.remove("check");
+    });
+    target.classList.add("check");
+}
+
+/**
+ * при виборі пункта меню показує відповідний текст
+ * author: Rostyk Broslavsky
+ * email: broslavsky.rostislaw@gmail.com
+ */
+function showStory(index) {
+    let textChild = Array.from(text.children);
```

Comment:
In general you shoul try to avoid using of `children`. Because you markup will change sometimes. For examples if you will add  element  inside your `text` element then will be in your array too and all will break. You can select only elements what you need for your manipulation. in this example text.querySelecotrAll('p')